src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -647,10 +692,15 @@ private void retrievePullRequests(final BitbucketSCMSourceRequest request) throw
             try {
                 // We store resolved hashes here so to avoid resolving the commits multiple times
                 for (final ChangeRequestCheckoutStrategy strategy : strategies.get(fork)) {
-                    String branchName = "PR-" + pull.getId();
-                    if (strategies.get(fork).size() > 1) {
-                        branchName = "PR-" + pull.getId() + "-" + strategy.name().toLowerCase(Locale.ENGLISH);
-                    }
+                    String branchName = BitbucketSCMSource.applyPRsNamingStrategy(
```

Comment:
same here for `branchName` variable name

Reply:
I changed it to `pullDisplayName` which seems to be more in line with the other var names.